services/buildbot/master/master.cfg

```diff
@@ -80,6 +80,8 @@ ubuntu16_04_slaves = ['rax-ubuntu1604-%d' % (i,) for i in range(1,4)]
 
 allpy27_slaves = fedora22_slaves + fedora23_slaves + ubuntu12_04_slaves + ubuntu16_04_slaves
 
+win_slaves = ['bot-glyph-6', 'rax-win2012r2-1']
```

Comment:
Yep. I will rename it. It was there as a reminder of my intention to unify .

I will remove and use a separate win2012r2 builder